src/framework/core/js/Fluid.js

```diff
@@ -855,6 +880,15 @@ var fluid = fluid || fluid_3_0_0;
         return isFinite(string) && ((string % 1) === 0) ? Number(string) : NaN;
     };
 
+    /**
+     * "Ponyfill" for Number.isInteger - can be removed once we drop support for IE11
+     * @param {Any} value - A value to be tested for being an integer
+     * @return {Boolean} `true` if the value is a Number and a finite integer
+     */
+    fluid.isInteger = function (value) {
```

Comment:
I think it's worthwhile to add some simple tests for this new function.

Reply:
added